script.js

```diff
@@ -0,0 +1,198 @@
+// Create section, nested in main (index.html )
+const main = document.getElementById('main');
+const featuredDancers = document.createElement('section');
+featuredDancers.className = 'main-section-dancers';
+featuredDancers.id = 'featureDancer';
+main.appendChild(featuredDancers);
+
+const featuredDancersDIV = document.createElement('div');
+featuredDancersDIV.className = 'main-section-dancers-content';
+featuredDancers.appendChild(featuredDancersDIV);
+
+const featuredDancersTitle = document.createElement('h3');
+featuredDancersTitle.className = 'main-section-dancers-title';
+featuredDancersTitle.textContent = 'Featured Dancers';
+featuredDancersDIV.appendChild(featuredDancersTitle);
+
+const featuredDancersTitleUnderline = document.createElement('p');
+featuredDancersTitleUnderline.className = 'footer-top-underline display-flex-center';
+featuredDancersDIV.appendChild(featuredDancersTitleUnderline);
+
+const featuredDancersTitleUnderlineSp = document.createElement('span');
+featuredDancersTitleUnderlineSp.className = 'footer-top-underline-span';
+featuredDancersTitleUnderlineSp.className = 'footer-top-underline-span';
+featuredDancersTitleUnderline.appendChild(featuredDancersTitleUnderlineSp);
+
+const cardUL = document.createElement('ul');
+cardUL.className = 'main-section-dancers-cards';
+cardUL.id = 'cards-show';
+featuredDancersDIV.appendChild(cardUL);
+
+// Generate cards in the created section, nested in main (index.html )
+function ProjectCard(imageLink, imageAltText, title, descriprion, textContent) {
+  this.image = {
+    link: imageLink,
+    altText: imageAltText,
+  };
+  this.title = title;
+  this.descriprion = descriprion;
+  this.textContent = textContent;
+}
+
+const dancers = [];
+
+dancers.push(
+  new ProjectCard(
+    'images/dancer-Clara.jpg',
+    'Picture of Dancer',
+    'Clara Rossa',
+    "Salsa teacher and owner of 'Dance Studio - Havana Barcelona'",
+    'In 2001 won a first place in World Latin Compettitioon for young dancers and have got the chance to travel around the world.',
+  ),
+);
+
+dancers.push(
+  new ProjectCard(
+    'images/dancer-Clara.jpg',
+    'Picture of Dancer',
+    'Clara Rossa',
+    "Salsa teacher and owner of 'Dance Studio - Havana Barcelona'",
+    'In 2001 won a first place in World Latin Compettitioon for young dancers and have got the chance to travel around the world.',
+  ),
+);
+
+dancers.push(
+  new ProjectCard(
+    'images/dancer-Clara.jpg',
+    'Picture of Dancer',
+    'Clara Rossa',
+    "Salsa teacher and owner of 'Dance Studio - Havana Barcelona'",
+    'In 2001 won a first place in World Latin Compettitioon for young dancers and have got the chance to travel around the world.',
+  ),
+);
+
```

Comment:
[OPTIONAL]

- Here as well instead of using same content repeatedly, you can use customized content to make your website realistic.